matter_server/server/server.py

```diff
@@ -62,13 +62,15 @@ def __init__(
         vendor_id: int,
         fabric_id: int,
         port: int,
-        primary_interface: str | None,
+        listen_addresses: list[str] | None = None,
```

Comment:
is it indended to have a list here ?

Reply:
Yes, you may listen on multiple addresses or address families. E.g. to explicitly listen to what is the default today you'd pass `--listen-address 0.0.0.0 --listen-address ::`.

Reply:
Yeah, see above, I overlooked the append. All good.